packages/next/src/bin/next.ts

```diff
@@ -154,13 +164,13 @@ program
       '-p, --port <port>',
       'Specify a port number on which to start the application.'
     )
-      .argParser(myParseInt)
+      .argParser(parseValidPositiveInteger)
       .default(3000)
       .env('PORT')
   )
   .option(
     '-H, --hostname <hostname>',
-    'Specify a hostname on which to start the application (default: 0.0.0.0).'
+    'Specify a hostname on which to start the application (default: localhost).'
```

Comment:
Where is this default set? Isn't that redundant since Yargs will print the actually configured default?

Reply:
I interpreted this as the default hostname Next.js will start on, not the default value of the argument (it doesn't have one).

Reply:
Do you mean (default: localhost) can be omitted entirely?

Reply:
I'd rather not change it if we don't know for sure. `localhost` and `0.0.0.0` are different. `localhost` would not allow reaching it if you know the IP of the host.

There's already too much going on in this PR so I'd rather we split this out.

Reply:
👍 Reverted this, will create an issue or discussion for this